server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -177,6 +178,23 @@ public DocsStats docStats() {
         }
     }
 
+    /**
+     * Returns the file sizes for the current commit
+     */
+    public Map<String, Long> getLastCommittedSegmentFileSizes() {
```

Comment:
Is there a reason we need to expose all the individual files of the commit? Could we instead have `long getLastCommitSizeInBytes()` that serves the narrow purpose of our needs?